yoda/llava_data_prep/requirements.txt

```diff
@@ -0,0 +1,120 @@
+aiohappyeyeballs==2.4.0
```

Comment:
Is this a pip freeze or the actual requirements (longer than our base-requirements.txt!)? If possible let's just add the requirements for the kit.